src/ia-clearable-text-input.ts

```diff
@@ -38,13 +40,21 @@ export class IaClearableTextInput extends LitElement {
    */
   @property({ type: Boolean }) focusOnClear = true;
 
+  /**
+   * If true, the clear button will be shown regardless of whether there is any
+   * text entered in the input field.
+   */
+  @property({ type: Boolean, reflect: true }) forceClearButton = false;
```

Comment:
q: can we do `displayClearButton = true` as default?

Reply:
I think defaulting boolean properties to true tends to be frowned upon, since they can't then be used like ordinary DOM attributes. (https://lit.dev/docs/components/properties/#boolean-attributes)

Note that the default behavior of this component is to show the clear button _only_ when there is text entered in the input field. The purpose of this new property is to _override_ that default and force the clear button to be shown at all times. I think `displayClearButton` might erroneously convey a toggle for whether the close button gets displayed at all, which is different from what this property does.